shared/biome.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "../packages/@biomejs/biome/configuration_schema.json",
+	"extends": ["shared/biome"],
+	"vcs": {
+		"clientKind": "git",
+		"enabled": true,
+		"useIgnoreFile": true
+	},
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true,
+			"style": {
+				"noNonNullAssertion": "off"
+			}
+		}
+	},
+	"organizeImports": {
+		"enabled": true
+	}
+}
```

Comment:
What the purpose of this file? An example?

Reply:
It's written in the description:
> I used our repository as a guinea pig. It means that the check command must pass.

I am dog-fooding the feature using our repository